fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/sink/writer/FlinkSinkWriterTest.java

```diff
@@ -0,0 +1,210 @@
+                                    .column("name", com.alibaba.fluss.types.DataTypes.CHAR(10))
+                                    .build())
+                    .build();
+
+    @ParameterizedTest
+    @ValueSource(strings = {"", "1"})
+    void testSinkMetrics(String clientId) throws Exception {
+        admin.createDatabase(
+                DEFAULT_SINK_TABLE_PATH.getDatabaseName(), DatabaseDescriptor.EMPTY, true);
+        createTable(DEFAULT_SINK_TABLE_PATH, TABLE_DESCRIPTOR);
+
+        Configuration flussConf = FLUSS_CLUSTER_EXTENSION.getClientConfig();
+        flussConf.set(ConfigOptions.CLIENT_ID, clientId);
+
+        InterceptingOperatorMetricGroup interceptingOperatorMetricGroup =
+                new InterceptingOperatorMetricGroup();
+        MockWriterInitContext mockWriterInitContext =
+                new MockWriterInitContext(interceptingOperatorMetricGroup);
+        FlinkSinkWriter flinkSinkWriter =
+                createSinkWriter(flussConf, mockWriterInitContext.getMailboxExecutor());
+
+        flinkSinkWriter.initialize(mockWriterInitContext.metricGroup());
+        flinkSinkWriter.write(
+                GenericRowData.of(1, StringData.fromString("a")), new MockSinkWriterContext());
+        flinkSinkWriter.flush(false);
+
+        Metric currentSendTime = interceptingOperatorMetricGroup.get(MetricNames.CURRENT_SEND_TIME);
+        assertThat(currentSendTime).isInstanceOf(Gauge.class);
+        // the default send latency is -1, so check it is >= 0, as the latency maybe very small 0ms
+        assertThat(((Gauge<Long>) currentSendTime).getValue()).isGreaterThanOrEqualTo(0);
+
+        Metric numRecordSend = interceptingOperatorMetricGroup.get(MetricNames.NUM_RECORDS_SEND);
+        assertThat(numRecordSend).isInstanceOf(Counter.class);
+        assertThat(((Counter) numRecordSend).getCount()).isGreaterThan(0);
+
+        flinkSinkWriter.close();
+    }
+
+    @Test
+    void testWriteExceptionWhenFlussUnavailable() throws Exception {
+        testExceptionWhenFlussUnavailable(
+                (writer, mailboxExecutor) -> {
+                    writer.getTableWriter().flush();
+                    assertThatThrownBy(
+                                    () ->
+                                            writer.write(
+                                                    GenericRowData.of(
+                                                            2, StringData.fromString("b")),
+                                                    new MockSinkWriterContext()))
+                            .hasRootCauseExactlyInstanceOf(NetworkException.class);
+                });
+    }
+
+    @Test
+    void testFlushExceptionWhenFlussUnavailable() throws Exception {
+        testExceptionWhenFlussUnavailable(
+                (writer, mailboxExecutor) -> {
+                    writer.getTableWriter().flush();
+                    assertThatThrownBy(() -> writer.flush(true))
```

Comment:
Add some comment for the tests that why the lines will throw exception, but `tableWriter#flush()` doesn't.